plugins/mac-virtualcam/src/camera-extension/OBSCameraDeviceSource.swift

```diff
@@ -267,7 +268,9 @@ class OBSCameraDeviceSource: NSObject, CMIOExtensionDeviceSource {
 
                 self._streamSink.stream.notifyScheduledOutputChanged(output)
             }
-            self.consumeBuffer(client)
+            DispatchQueue.main.asyncAfter(deadline: now + (1.0 / Double(OBSCameraFrameRate))) {
```

Comment:
To ensure that any available sample buffer is provided in time for the output frame rate, this should use twice the frame rate.

This is obviously a trade-off between frame pacing and CPU usage, but I'd rather check if CPU usage is low enough for good pacing at first.

Reply:
Modified to fire every `1.0 / (Double(OBSCameraFrameRate) * 2.0)` seconds.